config/utils/features.js

```diff
@@ -0,0 +1,15 @@
+const ProcessService = require('../../utils/process/process');
+
+function DetermineFeaturesFromEnvironment() {
+    const processService = ProcessService(process);
+    const features = processService.getEnvironmentValue('FEATURES')
```

Comment:
we don't have linting on this file but I still think we should folow the same rules so... missing semicolon

Reply:
make sense. We should do a couple of things in another Pull request.

- enable eslint for javascript
- fail the build when there is an error

Reply:
Done :)